draft-ietf-tls-tls13.md

```diff
@@ -751,6 +755,14 @@ or its successors.  Data to be signed/verified is hashed, and the
 result run directly through the ECDSA algorithm with no additional
 hashing.  The SignatureAndHashAlgorithm parameter in the DigitallySigned
 object indicates the digest algorithm which was used in the signature.
+Special signature-only curves MUST NOT be used for ECDSA unless otherwise
+noted.
+
+All EdDSA computations MUST be performed according to CFRG EdDSA document
+{{I-D.irtf-cfrg-eddsa}} or its successors. Data to be signed/verifies is
+passed as-is to the EdDSA algorithm with no hashing. And the signature
+output is placed as-is to the signature field. The SignatureAndHashAlgorithm
```

Comment:
s/And the/The/

s/to/in/

"SignatureAndHashAlgorithm.hash value must be set to none(0)"